components/selection/selection-controls.js

```diff
@@ -166,12 +181,8 @@ export class SelectionControls extends SelectionObserverMixin(RtlMixin(LocalizeC
 
 	_renderSelection() {
 		return html`
-			<d2l-selection-select-all></d2l-selection-select-all>
-			<d2l-selection-summary
-				aria-hidden="true"
```

Comment:
I removed `aria-hidden="true"` - I think this was being hidden because "Select all" was being repeated (aria-label on the select all checkbox, then again here). Since we're now displaying unique info here, I don't think we want it hidden.

Reply:
Makes sense!